Code of Conduct (#282)
Nuclear-Squid/ergol: CODE_OF_CONDUCT.md

Reviewer: moins de tolérance ~ne~ lui sera accordée qu’aux autres.*

> Code de conduite

Author: Pour que le plan et l’intention du document ne se perdent pas avec le temps, on peut les préciser en commentaire :

```suggestion
<!--
  - intro/résumé : ne soyez pas toxique
  - Comportement : qu’est-ce qui est considéré comme toxique
  - Modération : comment gère-t-on les comportements toxiques
  - Savoir-vivre : on ne se contente pas de ne pas être toxique, on essaye d’être de belles personnes.
  -->

Code de conduite
```

================================================================================

*TL;DR:*

> - Les règles ci-dessous peuvent se résumer à une phrase :
>   « **ne pas être toxique** ».
> - Si vous pensez qu’une discussion nécessite une intervention, contactez
>   l’équipe de modération.
> - Nous avons la chance d’avoir une communauté amicale, diverse et bienveillante.
>   Profitons-en, et cultivons cet esprit !

Author: Pour mieux refléter la structure logique du CoC on peut envisager d’ajouter des liens : 

```suggestion
- Les règles ci-dessous peuvent se résumer à une phrase :
  « **[ne soyez pas toxique][#comportement]** ».
- Si vous pensez qu’une discussion nécessite une intervention, contactez
  [l’équipe de modération][#modération].
- Nous avons la chance d’avoir une communauté amicale, diverse et bienveillante.
  Profitons-en, et [cultivons cet esprit][#savoir-vivre] !
```

Comportement
--------------------------------------------------------------------------------

1. Afin de conserver un environnement amical, sain et accueillant, nous ne
   tolérons aucun discours de haine ni comportement qui exclue les membres de
   groupes sociaux marginalisés : sexisme, hétérosexisme, racisme, xénophobie,
   transphobie, validisme, etc.
2. Nous vous prions aussi de faire preuve de gentillesse et de courtoisie.
   Traitez tout le monde avec respect.
3. Respectez le fait que les personnes ont des opinions différentes et que
   chaque choix d’ergonomie ou de disposition résulte de nombreux compromis qui
   peuvent être propres à chacun·e. Il y a rarement une bonne réponse pour tout
   le monde.
4. Veillez à éviter la critique non structurée et personnelle.
5. Peu importe qui vous êtes, si vous avez le sentiment d’avoir été harcelé·e ou
   mis·e dans une situation inconfortable par un membre de la communauté,
   contactez immédiatement l’équipe @Modération : nous tenons à faire de la

>    communauté un endroit sain, et nous sommes là pour vous aider.

Author: Ici, « endroit sain » est une mauvaise traduction de « safe place ». Quelqu’un a une idée de reformulation ?

Author: Dans la section d’après on utilise « à l’aise » et faute de mieux, je crois que ça pourrait faire l’affaire.
> nous tenons à ce que tout le monde se sente à l’aise, et nous sommes là pour vous aider.

Author: « à l’aise » m’irait bien, oui. Vendu.

   plus.
3. L’équipe de modération a toute autorité pour décider de la réponse à
   apporter, qui peut être un simple avertissement, une exclusion (temporaire ou
   permanente), ou toute autre action qui lui semblera pertinente pour préserver
   la communauté.
4. L’équipe de modération peut exiger la fin d’une conversation jugée trop
   offensante ou trop sensible. Continuer malgré la demande des modérateurices
   peut donner lieu à une sanction, voir le point 3.
5. Si l’équipe de modération sanctionne quelqu’un·e et si vous pensez que c’est
   injustifié, adressez-vous à une personne de cette équipe, **en privé**. Les
   plaintes sur le canal à propos des sanctions ne sont pas tolérées.
6. L’équipe de modération est tenue à un plus strict respect des règles que les
   autres membres de la communauté. Si un·e modérateurice cause une situation
   inappropriée, moins de tolérance lui sera accordée qu’aux autres.

L’équipe de modération est composée de [Nuclear Squid], [Ashenfae], [Trilowy],
[Nivopol].

Savoir-vivre
--------------------------------------------------------------------------------

- Les Ergonautes se préoccupent les un·e·s des autres. **Ne vous contentez pas

>   de ne pas être toxique : essayez d’être une bonne personne.**

Author: Suite à un échange avec @trilowy : 
```suggestion
  de ne pas être toxique : soyez la meilleure version de vous-même**, une
  personne agréable, avec qui on a plaisir à discuter.
```

Author: On pourrait aussi insérer ce « soyez la meilleure version de vous-même » dans le troisième item de l’introduction, pour contrebalancer le « ne soyez pas toxique ».

Author: En fait, « la meilleure version de vous-même » ça n’est pas neutre comme expression : https://www.marieclaire.fr/etre-meilleure-version-de-soi-injonction-effets-sante-mentale,1472388.asp

Comment peut-on reformuler ?

Reviewer 2: J’ai un souci avec "soyez la meilleure version de vous-même", c’est un red flag "développement personnel" pour moi (peut-être à tort).
Plus j’y pense, et plus je trouve que cette phrase « Ne vous contentez pas de ne pas être toxique : essayez d’être une bonne personne. » est une forme d’injonction et je ne sais pas quoi proposer comme reformulation. La phrase « Les Ergonautes se préoccupent les un·e·s des autres. » se suffirait presque à elle-même. Elle est positive et non stigmatisante. On peut même la mettre en gras, si on veut. Et les éléments qui suivent détaillent parfaitement le sens de ce qui est attendu.

Author: Quelque chose comme ça ?
> **Les Ergonautes se préoccupent les un·e·s des autres.** Soyez une personne agréable, avec qui on a plaisir à discuter.

Author: Ou si on veut sortir de l’injonction à « être » : 
> **Les Ergonautes se préoccupent les un·e·s des autres.** Comportez-vous de façon agréable, faites en sorte qu’on ait plaisir à discuter avec vous.

Author: Ou alors : 
> **Les Ergonautes se préoccupent les un·e·s des autres.** Soyez prévenants, faites en sorte qu’on ait plaisir à discuter avec vous.

Reviewer 3: Est-ce qu’on ne s’achemine vers la notion de « prendre soin » ? Ce serait une solution simple, compréhensible avec ou sans l’arrière-plan de l’éthique du *care*.

Author: Comme ça ?
>  Les Ergonautes prennent soin les un·e·s des autres. Comportez-vous de façon agréable, faites en sorte qu’on ait plaisir à discuter avec vous.

Reviewer 4: "Prendre soin" c’est un peu trop fort je trouve, dans le sens que c’est une grosse responsabilité de prendre soin de quelqu’un, alors que "préoccupe" véhiculait + l’idée de faire attention à respecter les autres. Enfin c’est peut-être juste que j’ai le sens des mots de travers.

Reviewer 3: Au contraire, pour moi, « se préoccuper de » est assez peu courant et plutôt neutre ou négatif (on pense d’ailleurs à « être préoccupé »), tandis que « prendre soin de » est positif, et plus actif que, par exemple, « faire attention à ». – L’essentiel est d’avoir un terme qui ne déplaise à personne : le sens est clair et la suite explicite l’idée.

Author: Vendu.

<!--
  - intro/résumé : ne soyez pas toxique !
  - Comportement : qu’est-ce qui est considéré comme toxique ?
  - Modération   : comment gère-t-on les comportements toxiques ?
  - Savoir-vivre : on ne se contente pas de ne pas être toxique,
                   on essaye d’être de belles personnes.
  -->

Code de conduite
================================================================================

*TL;DR:*

> - Les règles ci-dessous peuvent se résumer à une phrase :
>   « **aucun comportement toxique** ».
> - Si vous pensez qu’une discussion nécessite une intervention, contactez
>   l’équipe de modération.
> - Nous avons la chance d’avoir une communauté amicale, diverse et bienveillante.
>   Profitons-en, et cultivons cet esprit !

Author: Pour mieux refléter la structure logique du CoC on peut envisager d’ajouter des liens :
```suggestion
- Les règles ci-dessous peuvent se résumer à une phrase :
  « **[aucun comportement toxique][#comportement]** ».
- Si vous pensez qu’une discussion nécessite une intervention, contactez
  [l’équipe de modération][#modération].
- Nous avons la chance d’avoir une communauté amicale, diverse et bienveillante.
  Profitons-en, et [cultivons cet esprit][#savoir-vivre] !
```

--------------------------------------------------------------------------------

- Les Ergonautes se préoccupent les un·e·s des autres. **Ne vous contentez pas
  de ne pas être toxique : essayez d’être une bonne personne.**
- Évitez de jouer avec des sujets offensants ou sensibles, surtout si c’est hors
  sujet : trop souvent cela occasionne des disputes inutiles, blesse des
  personnes, abime la confiance mutuelle — parfois au point où des
  participant·e·s doivent quitter la communauté.
- Si quelqu’un·e relève quelque chose que vous avez dit ou fait, ne soyez pas
  sur la défensive : arrêtez simplement de faire ce qui vous a été reproché et
  présentez des excuses. Même si vous pensez avoir été mal interprété·e ou
  accusé·e injustement, il y a probablement quelque chose que vous auriez pu
  mieux communiquer — rappelez-vous que c’est à vous de mettre à l’aise les
  autres Ergonautes.
- Nous nous entendons bien, et nous sommes là pour parler de sujets
  enthousiasmants comme l’ergonomie, l’optimisation, et les belles choses qui
  font *clickety-clack*. Votre bonne foi sera un gage de confiance.

Les règles listées ci-dessus s’appliquent au serveur [Discord] ainsi qu’aux

> dépôts [Ergo‑L] et [OneDeadKey].

Author: Ce paragraphe devrait probablement aller en bas de la section « Comportement », non ?

Author: À moins que l’on préfère : 
> Ce code de conduite s’applique au serveur [Discord] ainsi qu’aux dépôts [Ergo‑L] et [OneDeadKey].

> Les règles listées ci-dessus s’appliquent au serveur [Discord] ainsi qu’aux
> dépôts [Ergo‑L] et [OneDeadKey].

Author: ```suggestion
Ce code de conduite s’applique au serveur [Discord] ainsi qu’aux dépôts [Ergo‑L]
et [OneDeadKey].
```

1. Les remarques qui enfreignent le code de conduite ne sont pas tolérées.
2. Les remarques que l’équipe @Modération considère inappropriées, qu’elles
   soient listées dans le code de conduite ou non, ne sont pas tolérées non
   plus.
3. L’équipe de modération a toute autorité pour décider de la réponse à
   apporter, qui peut être un simple avertissement, un /kick, un /mute, un /ban
   (temporaire ou permanent), ou toute autre action qui lui semblera pertinente
   pour préserver la communauté.
4. L’équipe de modération peut exiger la fin d'une conversation jugée trop
   offensante ou trop sensible. Continuer malgré la demande des modérateurices
   peut donner lieu à une sanction, voir le point 3.
5. Si un·e modérateurice sanctionne quelqu’un·e et vous pensez que c’est
   injustifié, adressez-vous à ce·tte modérateurice ou un·e autre, **en privé**.
   Les plaintes sur le canal à propos des sanctions ne sont pas tolérées.
6. L’équipe de modération est tenue à un plus haut standard que le reste des
   membres de la communauté. Si un·e modérateurice cause une situation
   inappropriée, moins de tolérance lui sera accordée qu’aux autres.

> 

Author: ```suggestion

Savoir-vivre
--------------------------------------------------------------------------------

```

>    inappropriée, moins de tolérance ne lui sera accordée qu’aux autres.

Reviewer 5: ```suggestion
   inappropriée, moins de tolérance lui sera accordée qu’aux autres.
```
je suis pas bien certain de comprendre. On est moins tolérant sur les modo car ils doivent être exemplaires ?

Reviewer 6: C’est ça

Author: Le `ne` a été ajouté par Astro, sans vouloir balancer. :-)
Et oui c’est l’idée de ce point, qui vient du CoC de Rust.

Code de conduite
================================================================================

*TL;DR:*

> - On a la chance d’avoir une communauté cordiale, diverse et bienveillante.

Author: ```suggestion
- Nous avons la chance d’avoir une communauté amicale, diverse et bienveillante.
```

> - Si vous vous comportez comme un connard, vous dégagez. Non négociable.

Author: ```suggestion
- Les règles ci-dessous peuvent se résumer à une phrase : « ne pas se comporter comme un connard ».
```

  « ne pas se comporter comme un connard ».
- Si vous pensez qu’une discussion nécessite une intervention, contactez

>   l'équipe de modération.

Reviewer 7: On peut aussi contacter les modos, enfin certains, via Github. À voir comment on pourrait avoir une sonnerie collective.

Comportement
--------------------------------------------------------------------------------

1. Afin de conserver un environnement amical, sain et accueillant, nous ne
   tolérons aucun discours de haine : remarque sexiste, hétéronormative,

>    raciste, xénophobe, validiste, offensant, etc. Nous ne tolérons pas non plus

Author: J’avoue m’être demandé si c’était inclus dans « hétéronormative » mais dans le doute, j’aurais dû le préciser, en effet.

>    raciste, xénophobe, validiste, offensant, etc. Nous ne tolérons pas non plus

Author: (et « offensante » tant qu’on y est, pour l’accord avec « remarque »)

>    tolérons aucun discours de haine : remarque sexiste, hétéronormative,
>    raciste, xénophobe, validiste, offensant, etc. Nous ne tolérons pas non plus

Author: ```suggestion
   tolérons aucun discours de haine : remarque sexiste, homophobe, transphobe,
   raciste, xénophobe, validiste, offensante, etc. Nous ne tolérons pas non plus
```
wdyt?

Reviewer 8: C’est mieux. « Hétéronormative » était bien aussi, et il porte un sens différent d’« homophobe »

Author: Finalement on va prendre la suggestion de @Ashenfae plus haut. :-) 
Merci !

Reviewer 6: Maintenant que j’ai compris comment ça marche 😇

   les comportements qui excluent les membres de groupes sociaux
   marginalisés.
2. Nous vous prions aussi de faire preuve de gentillesse et de courtoisie.
   Traitez tout le monde avec respect.
3. Respectez le fait que les personnes ont des opinions différentes et que

>    chaque choix de concept ou de disposition comporte de nombreux compromis et
>    différents coûts. Il y a rarement une bonne réponse pour tout le monde.

Author: ```suggestion
   chaque choix d’ergonomie ou de disposition résulte de nombreux compromis qui
   peuvent être propres à chacun·e. Il y a rarement une bonne réponse pour tout le monde.
```

4. Veillez à éviter la critique non structurée et personnelle.
5. Peu importe qui vous êtes, si vous avez le sentiment d’avoir été harcelé·e ou
   mis·e dans une situation inconfortable par un membre de la communauté, contactez
   immédiatement l'équipe @Modération : nous tenons à faire de la communauté un
   endroit sain, et nous sommes là pour vous aider.
6. De même, les comportements malveillants comme le spam, le troll, l'injure, la
   provocation, la condescendance, l’agressivité passive, la diffamation ou tout
   autre mécanisme d’accaparement de l'attention ne sont pas les bienvenus.
7. Pas d'autopromotion sauvage — ni en discussion publique, ni par message
   privé.

> 8. Il y a un canal dédié à la vente de matériel. L’utilisation de messages

Author: ```suggestion
8. Le serveur [Discord] dispose d’un canal dédié à la vente de matériel. L’utilisation de messages
```

   privés pour recommander ou vendre du matériel n’est pas tolérée et doit être
   signalée immédiatement à l’équipe @Modération.

> 9. <https://xkcd.com/1357/>

Author: On l’avait envisagé, mais on s’est dit que c’était mieux comme ça — à la fois plus complet et plus compact. Mais si tu sais nous introduire ça en une courte phrase c’est top !

> 9. <https://xkcd.com/1357/>

Reviewer 8: « Vous bénéficiez de la liberté d’expression. Mais nous aussi, et on ne vous doit pas une plateforme » ?

> 9. <https://xkcd.com/1357/>

Author: Pas fan. Je crois que le sujet est plus complexe que ça. Ce que j’ai trouvé de plus concis c’est ça : (node.js)

> **Your free speech is not our problem.** We recognize that this policy is a restrictive political stance. That is the intent. If you want a room with different rules, go create one.

Mais la phrase d’accroche est contre-intutive — j’entends déjà le tonton se plaindre de « la dictature bien-pensante », entre autres. Je pense que laisser un lien qui donne à réfléchir sur le sujet est plus efficace.

Ou alors, un lien de ce type ? (C’est moins geek, mais ça se discute.)

> [La liberté d’expression n’autorise personne à être une merde.](https://xkcd.com/1357/)

Modération
--------------------------------------------------------------------------------

1. Les remarques qui enfreignent le code de conduite ne sont pas tolérées.
2. Les remarques que l’équipe @Modération considère inappropriées, qu’elles
   soient listées dans le code de conduite ou non, ne sont pas tolérées non
   plus.

> 3. L’équipe de modération a toute autorité pour décider de la réponse à

Author: Ou alors, on ne précise pas les /kick /mute /ban en particulier et on reste général : 

> L’équipe de modération a toute autorité pour décider de la réponse à apporter, qui peut être un simple avertissement, une exclusion (temporaire ou permanente), ou toute autre action qui lui semblera pertinente pour préserver la communauté.

   apporter, qui peut être un simple avertissement, un /kick, un /mute, un /ban
   (temporaire ou permanent), ou toute autre action qui lui semblera pertinente
   pour préserver la communauté.
4. L’équipe de modération peut exiger la fin d'une conversation jugée trop
   offensante ou trop sensible. Continuer malgré la demande des modérateurices
   peut donner lieu à une sanction, voir le point 3.
5. Si l’équipe de modération sanctionne quelqu’un·e et si vous pensez que c’est

>    injustifié, adressez-vous à ce·tte modérateurice ou un·e autre, **en privé**.

Author: ```suggestion
   injustifié, adressez-vous à une personne de l’équipe, **en privé**.
```

   Les plaintes sur le canal à propos des sanctions ne sont pas tolérées.

> 6. L’équipe de modération est tenue à un plus haut standard que le reste des

Author: ```suggestion
6. L’équipe de modération est tenue à un plus strict respect des règles que le reste des
```

   membres de la communauté. Si un·e modérateurice cause une situation
   inappropriée, moins de tolérance lui sera accordée qu’aux autres.

Savoir-vivre
--------------------------------------------------------------------------------

Dans la communauté Ergo‑L, nous nous efforçons de nous occuper les uns des
autres. **Ne vous contentez pas de ne pas être un connard : essayez d’être une
bonne personne.** En particulier, évitez de jouer avec des sujets offensants ou

> sensibles, surtout si c’est hors sujet : cela mène trop souvent à des disputes
> inutiles, des sentiments heurtés et une confiance endommagée ; pire, cela peut

Author: ```suggestion
sensibles, surtout si c’est hors sujet : trop souvent cela occasionne des disputes
inutiles, blesse des personnes et abime la confiance mutuelle ; pire, cela peut
```

> inutiles, blesse des personnes et abime la confiance mutuelle ; pire, cela peut

Reviewer 9: Utiliser "pire" ici peut faire croire que blesser des personnes c'est moins grave que de faire partir des gens. je dirais "Cela peut _aussi_ faire fuir des gens"..

Author: Quand les gens s’en vont c’est très souvent parce que c’est la dernière limite mentale pour elleux. Mais oui, en effet, ce « pire » peut être mal interprété. Peut-être faudrait-il reformuler dans cet esprit : 

> trop souvent cela occasionne des disputes inutiles, blesse des personnes, abime la confiance mutuelle — parfois au point où des participant·e·s doivent quitter la communauté.

(En l’occurrence, c’est un cas assez concret qui s’est produit au sein de la commu Bépo. Notre commu y échappe jusqu’ici.)

> inutiles, blesse des personnes et abime la confiance mutuelle ; pire, cela peut
> faire fuir des gens de la communauté.

Author: ```suggestion
inutiles, blesse des personnes, abime la confiance mutuelle  — parfois au point
où des participant·e·s doivent quitter la communauté.
```

> Et si quelqu’un relève quelque chose que vous avez dit ou fait, ne soyez pas sur

Author: ```suggestion
Et si quelqu’un·e relève quelque chose que vous avez dit ou fait, ne soyez pas sur
```

> la défensive. Arrêtez simplement de faire ce qui vous a été reproché et

Reviewer 9: "prenez du recul sur la situation" au lieu de "ne soyez pas sur la défensive". On ne peut pas empêcher quelqu'un d'être sur la défensive, mais on peut les forcer à prendre du recul.

Author: Encore une fois, il y a possiblement une mauvaise traduction ici, mais il ne s’agit pas d’une prise de recul.

« Ne réagissez pas de manière défensive » a du sens ici : l’objet n’est pas d’ignorer ou de minimiser la demande de l’équipe de modération, mais de l’**accepter** et non de lutter contre (= comportement défensif). Il me semble que c’est la formulation appropriée en français mais bon, je ne suis ni psy ni lettré, donc un avis qualifié pourrait être utile ici.

Reviewer 9: ah, si c'est du feedback de modo dont on parles il faudrait être plus précis. Je pensais que c'était du feedback de n'importe qui, pas spécifiquement des modo envers quelqu'un qui dépasse le cadre

J'ai pas lu les originaux en anglais donc je lis ça comme la plupart des gens qui vont le lire, sans le contexte du document de référence

Reviewer 9: si le comportement est de ne pas argumenter sur le feedback c'est peut être mieux de le dire explicitement. C'est pas clair que "ne pas être sur la défensive" ça veut dire "on essaie pas de se justifier à un modo qui te fait une réflexion"

Author: Ni à un·e modo, ni au reste du groupe. Mais dans cette section-là (et par contraste aux autres) on essaye de mettre l’accent sur des *ambitions* positives : on n’est plus dans le règlement, mais dans l’incitation.

présentez des excuses. Même si vous pensez avoir été mal interprété·e ou accusé·e
injustement, il y a probablement quelque chose que vous auriez pu mieux
communiquer — rappelez-vous que c’est à vous de mettre à l’aise les autres
Ergonautes. Tout le monde veut bien s’entendre et nous sommes ici pour parler de
choses enthousiasmantes comme l’ergonomie, l’optimisation, et les belles choses
qui font *clickety-clack*. Vous verrez que les gens accepteront votre bonne foi et
vous pardonneront si vous faites en sorte de gagner leur confiance.

Les règles listées ci-dessus s’appliquent au serveur [Discord]
ainsi qu’aux dépôts [Ergo‑L] et [OneDeadKey].

> *(Adapté du [Code de conduite Rust][Rust], de la [Node.js Policy on
> Trolling][nodejs] et du [Contributor Covenant v1.3.0][cco].)*

Author: ```suggestion
*(Sources d’inspiration : [Code de conduite Rust][Rust], [Node.js Policy on
Trolling][nodejs] et [Contributor Covenant v1.3.0][cco].)*
```

Code de conduite
================================================================================

*TL;DR:*
- Nous avons la chance d’avoir une communauté amicale, diverse et bienveillante.
- Les règles ci-dessous peuvent se résumer à une phrase :
  « ne pas se comporter comme un connard ».
- Si vous pensez qu’une discussion nécessite une intervention, contactez

>   l'équipe de modération.

Author: Plus qu’une intro, c’est la version courte du document, avec les trois points essentiels (à mes yeux).

Je trouve ça beaucoup plus lisible sous la forme d’une liste, mais c’est peut-être lié à un piètre niveau de lecture et de rédaction de ma part. Génération PowerPoint ?

>   l'équipe de modération.

Author: En fait on pourrait retomber sur quelque chose qui colle au plan avec cette permutation : 

> - Les règles ci-dessous peuvent se résumer à une phrase : « ne pas se comporter comme un connard ».
> - Si vous estimez qu’une discussion nécessite une intervention, contactez l'équipe de modération.
> - Nous avons la chance d’avoir une communauté amicale, diverse et bienveillante. Profitons-en, et cultivons cet esprit !

wdyt?

>   l'équipe de modération.

Reviewer 3: 👌

>   l'équipe de modération.

Author: Le camarade AntoineD vient de faire mon éducation : https://www.piokok.com/profile/cestquoicetteinsulte/
![pt2_6833467160341013376458_9_a7018a42fe5f659484ceeb71e1a1a5e5](https://github.com/user-attachments/assets/e0345089-2d8a-4ae9-877e-9ba8e9cb55db)
![pt2_6851695010384473357245_4_fd8db3401d2a9455280820deb07d6aa6](https://github.com/user-attachments/assets/59d79727-2b84-4ea8-8352-23b316aa89dd)

>   l'équipe de modération.

Author: J’aime bien « raclure », perso.
J’aime aussi « sac à merde », qui me rappelle un grand film d’auteur (*hasta la vista*).
Je trouve « ordure » un peu vieillot/théâtral, mais ça joue aussi.

>   l'équipe de modération.

Author: On me signale que « crevure » serait choupi également.

>   l'équipe de modération.

Author: J’ai utilisé « ne pas être toxique » pour l’instant. On choisira le vocabulaire plus tard.

Comportement
--------------------------------------------------------------------------------

> 1. Afin de conserver un environnement amical, sain et accueillant, nous ne

Author: La répétition ne me choque pas. Le « sain » du point 5 est une mauvaise traduction de « safe », et pourrait être reformulé.

   tolérons aucun discours de haine : remarque sexiste, hétéronormative,
   raciste, xénophobe, validiste, offensant, etc. Nous ne tolérons pas non plus

>    les comportements qui excluent les gens dans des groupes sociaux

Author: Pas fan de l’« appartenance » à un groupe. Peut-être « faisant partie » (mais c’est possiblement pareil) ou « issus de » (mais c’est possiblement idiot)  ?

>    les comportements qui excluent les gens dans des groupes sociaux

Author: ```suggestion
   les comportements qui excluent les membres de groupes sociaux
```

   marginalisés.

> 2. Nous vous prions aussi d’être gentil et courtois. Traitez tout le monde avec

Author: ```suggestion
2. Nous vous prions aussi de faire preuve de gentillesse et de courtoisie. Traitez tout le monde avec
```

>    respect.

Author: ```suggestion
   Traitez tout le monde avec respect.
```

> 1. Afin de conserver un environnement amical, sain et accueillant, nous ne
>    tolérons aucun discours de haine ni comportement qui exclue les gens dans des groupes sociaux
>    marginalisés : sexisme, hétérosexisme, racisme, xénophobie, transphobie, validisme, etc.
>    raciste, xénophobe, validiste, offensant, etc. Nous ne tolérons pas non plus
>    les comportements qui excluent les membres de groupes sociaux
>    marginalisés.

Author: ```suggestion
1. Afin de conserver un environnement amical, sain et accueillant, nous ne
   tolérons aucun discours de haine ni comportement qui exclue les membres
   de groupes sociaux marginalisés : sexisme, hétérosexisme, racisme, xénophobie,
   transphobie, validisme, etc.
```

3. Respectez le fait que les personnes ont des opinions différentes et que

>    chaque choix de concept ou de disposition comporte de nombreux compromis et

Author: « dépenses » ne serait pas correct ici : « coûts » est à prendre au sens de contre-partie d’un choix.
(Encore une fois on hérite de la mauvaise traduction du CoC Rust.)

>    chaque choix de concept ou de disposition comporte de nombreux compromis et

Reviewer 3: Je vois ce que tu veux dire. C’est surtout la syntaxe qui coinçait…
 
« chaque choix de concept ou de disposition résulte de compromis et a un coût qui lui est propre » ?

Je trouve aussi « concept » un peu vague ici.

>    chaque choix de concept ou de disposition comporte de nombreux compromis et

Reviewer 3: Avec « être », sans « s », c’est parfait !

   différents coûts. Il y a rarement une bonne réponse pour tout le monde.

> 4. Veillez à éviter la critique non structurée et personnelle autant que
>    possible.

Author: ```suggestion
4. Veillez à éviter la critique non structurée et personnelle.
```

5. Peu importe qui vous êtes, si vous avez le sentiment d’avoir été harcelé·e ou

>    mis·e en situation inconfortable par un membre de la communauté, contactez

Author: ```suggestion
   mis·e dans une situation inconfortable par un membre de la communauté, contactez
```

   immédiatement l'équipe @Modération : nous tenons à faire de la communauté un
   endroit sain, et nous sommes là pour vous aider.
6. De même, les comportements malveillants comme le spam, le troll, l'injure, la
   provocation, la condescendance, l’agressivité passive, la diffamation ou tout

>    autre mécanisme d’accaparement d'attention ne sont pas les bienvenus.

Author: ```suggestion
   autre mécanisme d’accaparement de l'attention ne sont pas les bienvenus.
```

7. Pas d'autopromotion sauvage — ni en discussion publique, ni par message
   privé.
8. Il y a un canal dédié à la vente de matériel. L’utilisation de messages
   privés pour recommander ou vendre du matériel n’est pas tolérée et doit être
   signalée immédiatement à l’équipe @Modération.
9. <https://xkcd.com/1357/>

Modération
--------------------------------------------------------------------------------

1. Les remarques qui enfreignent le code de conduite ne sont pas tolérées.
2. Les remarques que l’équipe @Modération considère inappropriées, qu’elles
   soient listées dans le code de conduite ou non, ne sont pas tolérées non
   plus.
3. L’équipe de modération a toute autorité pour décider de la réponse à
   apporter, qui peut être un simple avertissement, un /kick, un /mute, un /ban
   (temporaire ou permanent), ou toute autre action qui lui semblera pertinente
   pour préserver la communauté.
4. L’équipe de modération peut exiger la fin d'une conversation jugée trop
   offensante ou trop sensible. Continuer malgré la demande des modérateurices
   peut donner lieu à une sanction, voir le point 3.

> 5. Si un·e modérateurice sanctionne quelqu’un·e et vous pensez que c’est

Author: Là je pige pas trop ce « que » supplémentaire. Ça impliquerait qu’on dise « si que… » , non ?

> 5. Si un·e modérateurice sanctionne quelqu’un·e et vous pensez que c’est

Author: Pas compris. Concrètement, quels passages faudrait-il modifier ?

> 5. Si un·e modérateurice sanctionne quelqu’un·e et vous pensez que c’est

Reviewer 3: Incompréhension réciproque ;-) En gardant la syntaxe, le « que » s’impose (ou, moins bien, une répétition du « si ») : <https://fr.wiktionary.org/wiki/si> (mais le subjonctif ne colle pas au contexte).

> 5. Si un·e modérateurice sanctionne quelqu’un·e et vous pensez que c’est

Reviewer 3: Ici, on a un « quelqu’un·e » et plus loin un « quelqu’un ». À mon avis, tout est possible (même si « ce·tte modérateurice » ne me plaît pas, avec un point médian puis une forme synthétique), mais il faut harmoniser. Après, sur le serveur, la liberté règne, bien sûr.

> 5. Si un·e modérateurice sanctionne quelqu’un·e et vous pensez que c’est

Author: OK donc je ne comprends absolument pas. Écris la phrase complète STP ?
Répéter le « si » me choque tellement moins que ce « si… que ».

> 5. Si un·e modérateurice sanctionne quelqu’un·e et vous pensez que c’est

Reviewer 3: « Si l’équipe de modération sanctionne une personne et que vous pensez que c’est injustifié, adressez-vous » : formulation correcte, neutre et courante pour moi (et par exemple [ici](https://www.languefrancaise.net/forum/viewtopic.php?id=10723)).

> 5. Si un·e modérateurice sanctionne quelqu’un·e et vous pensez que c’est

Author: ```suggestion
5. Si l’équipe de modération sanctionne quelqu’un·e et si vous pensez que c’est
```

> 5. Si un·e modérateurice sanctionne quelqu’un·e et vous pensez que c’est

Author: J’ai opté pour le double « si », conformément à cette remarque dans le lien que tu as cité :
> Cela dépend de l'époque à laquelle vous vivez :
> si vous vivez au XVII e siècle, vous écrirez si... et que...
> Si vous vivez au XXIe siècle, vous écrirez si... et si...

(Le « si… et que… » m’écorche vraiment les yeux. C’est probablement trop soutenu pour moi.)

   injustifié, adressez-vous à ce·tte modérateurice ou un·e autre, **en privé**.
   Les plaintes sur le canal à propos des sanctions ne sont pas tolérées.
6. L’équipe de modération est tenue à un plus haut standard que le reste des
   membres de la communauté. Si un·e modérateurice cause une situation
   inappropriée, moins de tolérance lui sera accordée qu’aux autres.

Savoir-vivre
--------------------------------------------------------------------------------

> Dans la communauté Ergo‑L, nous nous efforçons de nous occuper les uns des

Author: Oui, c’est volontaire.
Et pourquoi faudrait-il une liste numérotée ici mais pas dans le résumé en début de fichier ?

> Dans la communauté Ergo‑L, nous nous efforçons de nous occuper les uns des

Reviewer 3: Je vois les différentes rubriques comme des parties sur le même plan. L’intro est d’une autre nature, et n’a pas tout à fait le même ton. – Mais l’homogénéité des rubriques me semble plus importante que la forme de l’intro.

> Dans la communauté Ergo‑L, nous nous efforçons de nous occuper les uns des

Author: Les numéros dans ces deux sections servent à identifier des règles — soit pour qualifier une faute d’un·e Ergonaute, soit pour se référer à une action de modération.

La dernière partie du document se veut plus informelle, pour expliquer l’esprit du CoC et des attentes que l’on a envers les Ergonautes. Je pense donc que c’est un avantage de rompre avec la présentation sous forme de liste numérotée dans cette section.

> Dans la communauté Ergo‑L, nous nous efforçons de nous occuper les uns des

Reviewer 3: Très bien.

Dans ce cas, peut-être marquer la différence dans les titres ? En tout cas, c’est bon pour moi.

> autres. **Ne vous contentez pas de ne pas être un connard : essayez d’être une

Reviewer 3: Dit comme ça, je comprends mieux et ça me paraît clair.

D’accord sur le fait que la modération a le dernier mot sur le fond et la forme !

bonne personne.** En particulier, évitez de jouer avec des sujets offensants ou

> sensibles, surtout si c’est hors-sujet : cela mène trop souvent à des disputes

Author: ```suggestion
sensibles, surtout si c’est hors sujet : cela mène trop souvent à des disputes
```

> inutiles, des sentiments heurtés et une confiance endommagée ; pire, cela peut

Author: Oui la traduction francophone du CoC Rust est horrible. Merci.

faire fuir des gens de la communauté.

Et si quelqu’un relève quelque chose que vous avez dit ou fait, ne soyez pas sur
la défensive. Arrêtez simplement de faire ce qui vous a été reproché et

> excusez-vous. Même si vous pensez avoir été mal interprété·e ou accusé·e

Author: ```suggestion
présentez des excuses. Même si vous pensez avoir été mal interprété·e ou accusé·e
```

injustement, il y a probablement quelque chose que vous auriez pu mieux
communiquer — rappelez-vous que c’est à vous de mettre à l’aise les autres

> Ergonautes. Tout le monde veut bien s’entendre et nous sommes ici pour parler de

Author: OK la traduction initiale est naze, mais cette proposition me semble un peu trop ampoulée, pas assez naturelle.

Peut-être faut-il formuler cette phrase différemment (voire dans un nouveau paragraphe), en partant du constat qu’on ne veut que parler de clavier et d’ergonomie, et donc qu’on gagne à le faire en entretenant de bonnes relations entre nous ?

(Toute cette dernière partie « Savoir-vivre » me semble mal rédigée, en fait.)

> Ergonautes. Tout le monde veut bien s’entendre et nous sommes ici pour parler de

Reviewer 3: Je n’ai pas comparé avec les textes qui ont servi de base et j’ai proposé le moins de réécriture possible. Il faut peut-être reprendre un peu plus en profondeur pour adapter, oui.

> Ergonautes. Tout le monde veut bien s’entendre et nous sommes ici pour parler de

Author: J’ai reformaté toute la dernière section. Merci pour toutes les remarques, je crois qu’elle est bien plus lisible à présent.

choses enthousiasmantes comme l’ergonomie, l’optimisation, et les belles choses

> qui font clickety-clack. Vous verrez que les gens accepteront votre bonne foi et

Author: ```suggestion
qui font *clickety-clack*. Vous verrez que les gens accepteront votre bonne foi et
```

vous pardonneront si vous faites en sorte de gagner leur confiance.

> Les règles d’application listées ci-dessus s’appliquent au serveur [Discord]

Author: Oh bon sang. Oui.
